templates/index.html

```diff
@@ -0,0 +1,74 @@
+<html>
+<head>
+    <title> Index of {{ path }} </title>
+    <link rel="stylesheet" href="{{url_for('static', filename='/common/css/fileindex.css')}}" type="text/css">
+</head>
+<body>
+    <h1> Index of {{ path }} </h1>
+    <table id="indexlist">
+        <tr class="indexhead">
```

Comment:
Maybe I don't understand something, but where did you define the classes "indexhead", "indexcolicon", "indexcolname" ...

Reply:
Dropped.